### ISSUE TITLE
Deprecation warning from flatten

### ISSUE THREAD

**Issue by shernshiou** [jeromew/node-pg-copy-streams-binary#3]
Hi,

I am using Slonik (https://github.com/gajus/slonik/) which depends on this package. One of your dependencies (flatten) produces such warning:

```
pg-copy-streams-binary > flatten@1.0.2: I wrote this module a very long time ago; you should use something else.
```

If you need help updating the package, I can help. 

Thanks.

**Comment by jeromew** [comment]
Thanks for the report.
I agree that we can get rid of this dependency. I'll review your PR shortly.

**Comment by jeromew** [comment]
published 1.2.0 as a result. thanks for the PR.
